packages/roosterjs-editor-plugins/lib/BlockQuote/BlockQuote.ts

```diff
@@ -0,0 +1,75 @@
+import { getBlockQuoteElement, getNodeAtCursor } from 'roosterjs-editor-api';
+import { Editor, EditorPlugin } from 'roosterjs-editor-core';
+import {
+    PluginDomEvent,
+    PluginEvent,
+    PluginEventType,
+    ContentScope,
+    ContentPosition,
+} from 'roosterjs-editor-types';
+
+const KEY_ENTER = 13;
+
+// An editor plugin to handle blockquote event
+export default class BlockQuote implements EditorPlugin {
```

Comment:
How to trigger this plugin?

Reply:
If user press enter on an empty line of blockquote, user will exit the blockquote and start a new line.